docker/run.sh

```diff
@@ -0,0 +1,45 @@
+#! /bin/sh
+
+set -e
+BASE_URL=${BASE_URL:-/}
+NGINX_ROOT=/usr/share/nginx/html
+INDEX_FILE=$NGINX_ROOT/index.html
+NGINX_CONF=/etc/nginx/nginx.conf
+
+node /usr/share/nginx/configurator $INDEX_FILE
+
+# TODO: this is empty but we'll be adding configuration values here
+
+if [[ "${BASE_URL}" != "/" ]]; then
+  sed -i "s|location / {|location $BASE_URL {|g" $NGINX_CONF
+fi
+
+## Adding env var support for swagger file (json or yaml)
+if [[ -f "$SWAGGER_FILE" ]]; then
```

Comment:
I have not yet tested whether this preserves or breaks this `docker run -d -p 80:8080 -v $(pwd):/tmp -e SWAGGER_FILE=/tmp/swagger.json swaggerapi/swagger-editor` functionality.